src/components/CommentList/CommentList.tsx

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
 import './CommentList.scss';
+import { Comment } from '../../types';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  comment: Comment[];
+};
+
+export const CommentList: React.FC<Props> = ({ comment }) => (
+  <>
```

Comment:
no need for a fragment here
<ul> does the wrapping - so your JSX returns one html element
```suggestion
```

Reply:
ofc remove the closing tag as well